packages/nimble-components/src/table-column/text/template.ts

```diff
@@ -2,11 +2,18 @@ import { html } from '@microsoft/fast-element';
 import type { TableColumnTextCellRecord, TableColumnTextColumnConfig } from '.';
 import type { TableCellState } from '../../table/types';
 
+const setTitleWhenOverflow = (span: HTMLElement): void => {
+    if (span?.textContent?.trim() && span.offsetWidth < span.scrollWidth) {
+        span.setAttribute('title', span.textContent.trim());
```

Comment:
Instead of pulling the textContent from the view, can we have a shared function that resolves the text content from the cellstate. i.e. make one of these shared functions that takes the cellstate and calculates the content instead of serializing it from the view.

Reply:
Added a function `getCellContent`.